vignettes/Carrying-capacity.Rmd

```diff
@@ -81,17 +81,9 @@ lines(s_seasonal$pfpr ~ s_seasonal$timestep, col = "darkorchid3")
 ## Custom modification of the mosquito carrying capacity
 
 In malariasimulation, we can also modify the carrying capacity over time in a more
-bespoke manner. Firstly it is helpful to start with our baseline (equilibrium)
-carrying capacity. We can access this for our baseline parameters with:
-
-```{r}
-cc <- get_init_carrying_capacity(p)
-cc
-```
-
-We can then use this in combination with `set_carrying_capacity()` to provide
-modified, species-specific carrying capacity values at given timepoints. This
-allows us to model a number of useful things:
+bespoke manner. We can use the `set_carrying_capacity()` helper function to provide
+scalers to modify the species-specific carrying capacity values at given time points.
```

Comment:
I know it states it in the functions documentation, but might it be useful to clarify here too that each scalar the user specifies through time acts on the baseline carrying capacity rather than the previous scaled carrying capacity? Also, probably pedantic, but is it worth saying "time steps" rather than "time points" just for consistency of language/names? Perhaps:

"We can use the `set_carrying_capacity()` helper function to specify scalars that modify the species-specific carrying capacity through time, relative to the baseline carrying capacity, at defined time steps."?